bpf/mock/conntrack_vacant.h

```diff
@@ -0,0 +1,15 @@
+/* SPDX-License-Identifier: GPL-2.0 */
```

Comment:
This file is now generated, correct? Can you inject a comment in these generated files? something like:
`/* GENERATED BY ... */`

Reply:
This is not generated. For customized functions, developers need to create this header manually and choose which functions to be mocked.